tpm2tools/seal_test.go

```diff
@@ -0,0 +1,238 @@
+package tpm2tools
+
+import (
+	"bytes"
+	"crypto/sha256"
+	"fmt"
+	"testing"
+
+	"github.com/google/go-tpm/tpm2"
+	"github.com/google/go-tpm/tpmutil"
+
+	"github.com/google/go-tpm-tools/internal"
+)
+
+func TestSeal(t *testing.T) {
+	rwc := internal.GetTPM(t)
+	defer rwc.Close()
+
+	key, err := StorageRootKeyRSA(rwc)
```

Comment:
We do these two operations a bunch (GetTPM + StorageRootKeyRSA), might be good as a helper function.

We also need to make sure we are always calling key.Close() to prevent handle depletion. Something like:
```go
func getSRK(tb testing.TB) (io.ReadWriteCloser, *Key) {
        tb.Helper()
        // ...
}
func TestSeal(t *testing.T) {
	rwc, srk := getSRK(t)
	defer rwc.Close()
        defer srk.Close()
        // ...
}
```

Reply:
The tests are currently failing on real hardware, I think the handle leak might be why.

Reply:
Fixed the handle leak.

Reply:
Something feels off about combining those in to one function.
I'd still like to catch the handle leaks in some way other than hardware testing. Once the activeHandles method goes in from Sam's patch, we can query active handles on simulator.Close() and fail the test.

Reply:
Follow up PR will have handle leak checking.